tests/Issues/AutoImport/Fixture/skip_conflict_case_insensitive_class_name.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
this fixture seems overlapped copied running test locally that not cleared with `.php`  extension, can be removed

```
rm tests/Issues/AutoImport/Fixture/skip_conflict_case_insensitive_class_name.php
```